rpcs3/rpcs3qt/save_manager_dialog.cpp

```diff
@@ -83,7 +83,7 @@ namespace
 				u32 iconSize = icon.size();
 				std::vector<uchar> iconData;
 				icon.read(iconData, iconSize);
-				save_entry2.iconBuf = iconData;
+				save_entry2.iconBuf = std::move(iconData);
```

Comment:
maybe just use the iconBuf in icon.read instead of using iconData in the first place

Reply:
Right, didn't notice it. Fix it.
https://github.com/RPCS3/rpcs3/pull/9346/commits/3460468fed521ca30f173178e1ddfc1cc2dc8a1f